benchmark/child_process/child-process-exec-stdout.js

```diff
@@ -1,41 +1,42 @@
 'use strict';
 const common = require('../common.js');
+const { exec, execSync } = require('child_process');
+const isWindows = process.platform === 'win32';
 
 var messagesLength = [64, 256, 1024, 4096];
-// Windows does not support that long arguments
-if (process.platform !== 'win32')
-  messagesLength.push(32768);
-const bench = common.createBenchmark(main, {
+// Windows does not support arguments that long
```

Comment:
Just a suggestion, feel free to ignore: I would replace `arguments` with `command lines` as Windows does not really have a concept of arguments. You could even be more specific, `Windows does not support command lines longer than 8191 characters` (there is no reason to believe that Microsoft is going to raise that limit).